catalog/spatial/wfs/1.1.0/spatial-wfs-v1_1_0-source/src/main/java/org/codice/ddf/spatial/ogc/wfs/v110/catalog/source/WfsFilterDelegate.java

```diff
@@ -280,142 +283,171 @@ private Boolean isFilterSet(FilterType filter) {
   @Override
   public FilterType propertyIsEqualTo(
       String propertyName, String literal, boolean isCaseSensitive) {
-    return buildPropertyIsFilterType(propertyName, literal, PROPERTY_IS_OPS.PropertyIsEqualTo);
+    return buildPropertyIsFilterType(
+        propertyName, literal, PROPERTY_IS_OPS.PropertyIsEqualTo, isCaseSensitive);
   }
 
   @Override
   public FilterType propertyIsEqualTo(String propertyName, Date date) {
     return buildPropertyIsFilterType(
-        propertyName, convertDateToIso8601Format(date), PROPERTY_IS_OPS.PropertyIsEqualTo);
+        propertyName,
+        convertDateToIso8601Format(date),
+        PROPERTY_IS_OPS.PropertyIsEqualTo,
+        DEFAULT_MATCH_CASE);
   }
 
   @Override
   public FilterType propertyIsEqualTo(String propertyName, int literal) {
-    return buildPropertyIsFilterType(propertyName, literal, PROPERTY_IS_OPS.PropertyIsEqualTo);
+    return buildPropertyIsFilterType(
+        propertyName, literal, PROPERTY_IS_OPS.PropertyIsEqualTo, DEFAULT_MATCH_CASE);
```

Comment:
could we use overloading here for buildPropertyIsFilterType so that the method uses the `DEFAULT_MATCH_CASE` if it isn't specified?